app/src/main/res/xml/appfilter.xml

```diff
@@ -1136,9 +1195,14 @@
     <!-- Lawnchair -->
     <item component="ComponentInfo{ch.deletescape.lawnchair.plah/ch.deletescape.lawnchair.Launcher}" drawable="lawnchair" />
     <item component="ComponentInfo{ch.deletescape.lawnchair.plah/ch.deletescape.lawnchair.LawnchairLauncher}" drawable="lawnchair" />
+    <item component="ComponentInfo{launcher.libre.dev/ch.deletescape.lawnchair.LawnchairLauncher}" drawable="librechair"/>
```

Comment:
Doesn't look like you added an icon for `librechair` or `librechairsettings`, did you mean to use the same ones as Lawnchair for those two?

Reply:
Yea Librechair is a fork of lawnchair with basically the same icons.